.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: ruby
 rvm:
-  - "2.5.3"
-  - "2.6.3"
+  - "2.5.8"
+  - "2.6.6"
+  - "2.7.1"
```

Comment:
[note] I've updated the CI configuration because I'd like to confirm whether the CI succeeds with the latest Ruby versions, but this change is not essentially related to this pull request.
So, I will revert it if you hope.